sdc/tests/test_dataframe.py

```diff
@@ -810,6 +810,19 @@ def test_impl(df):
                            'D': [None, 'dd', '', None]})
         pd.testing.assert_frame_equal(hpat_func(df), test_impl(df))
 
+    def test_df_isna(self):
+        def test_impl(df):
+            return df.isna()
+        sdc_func = sdc.jit(test_impl)
+        for idx in [[3, 4, 2, 6, 1], ['a', 'b', 'c', 'd', 'e'], None]:
```

Comment:
That's bad due to inability to comment particular indexes during debugging. I would use:
```
        sdc_func = sdc.jit(test_impl)
        df_indexes = [
               [3, 4, 2, 6, 1],
               ['a', 'b', 'c', 'd', 'e'],
               None
        ]
        for idx in df_indexes:
```